theme/templates/pages/download.html

```diff
@@ -80,9 +28,9 @@ <h1><a class="headline-link" href="#download">Download Mixxx</a></h1>
   </p>
 
   <p class="grid-container full">
-    {% with stable_version=page.stable_version %}
-    <a href="#stable" class="button button-primary download-button" title="{% trans %}Download stable version of Mixxx{% endtrans %}">{% trans %}Download {{ stable_version }}{% endtrans %}</a>
-    <span class="download-button-description">{% trans %}Select your Operating System below.{% endtrans %}</span>
+    {% with stable_version=page.versions.stable.name, stable_link="#stable" %}
+    <a href="#stable" class="button button-primary download-button" title="{% trans %}Download stable version of Mixxx{% endtrans %}" data-osdetect-text="{% trans %}Download Mixxx {{ stable_version }} for %s{% endtrans %}">{% trans %}Download Mixxx {{ stable_version }}{% endtrans %}</a>
+    <span class="download-button-description" data-osdetect-text="{% trans %}Want to download Mixxx for a different OS? Then select it manually &lt;a href=&quot;{{ stable_link }}&quot;&gt;below&lt;/a&gt;.{% endtrans %}">{% trans %}Select your Operating System <a href="{{ stable_link }}">below</a>.{% endtrans %}</span>
```

Comment:
Remove this line. It adds nothing.

Reply:
I think it is useful, at least when visiting the download page on Windows/macOS:

![Screenshot from 2020-12-30 23-26-46](https://user-images.githubusercontent.com/1834516/103384526-05d2a300-4af7-11eb-85dc-b6e3124c77c0.png)